### PR TITLE
Pcm 1266

### PR DESCRIPTION
**Pull request by jensengar**

### CODE REVIEW

**Comment by djhouseknecht** [comment]
Tests are going to run so fast without that logging integration test 🔥⚡️

File: src/client.ts [hunk 1]
```diff
@@ -73,15 +72,8 @@ export class PureCloudWebrtcSdk extends WildEmitter {
...
-  _logBufferSize: number;
-  _logTimer: NodeJS.Timeout | null;
   _connected: boolean;
   _streamingConnection: StreamingClient;
-  _backoffActive: boolean;
-  _failedLogAttempts: number;
-  _backoff: any;
   _orgDetails: any;
   _personDetails: IPersonDetails;
   _clientId: string;
```
[the remark this answers is not in the document]
Comment (jensengar) [reply src/client.ts:79]: nope. i'll remove it.

File: test/unit/client.integration.test.ts [hunk 2]
```diff
@@ -69,40 +80,29 @@ describe('Client', () => {
       }
     });
 
-    test('warns if environment is not valid', () => {
+    it('warns if environment is not valid', () => {
```
[the remark this answers is not in the document]
Comment (jensengar) [reply test/unit/client.integration.test.ts:83]: personally i prefer `it`. I'm not sure `ftest` works, but `fit` allows you to just run a single test which I use all the time.

[the remark this answers is not in the document]
Comment (djhouseknecht) [reply test/unit/client.integration.test.ts:83]: That makes sense. test has `test.only` which isn't as quick to type as `fit`. I will switch it up to start using `it`

**Pull request merged**